resources/js/Components/Modal/EventRegistrationModal.tsx

```diff
@@ -65,16 +68,25 @@ const EventRegistrationModal = ({
         wasSuccessful,
         processing,
     } = useForm<FormDataType>(SESSION_REGISTRATION_FORM_INITIAL_VALUE)
+    const {enqueueSnackbar} = useSnackbar()
 
     useEffect(() => {
         if (wasSuccessful) {
             clearErrors()
-            onClose()
+            if (onClose) {
```

Comment:
why do we need this btw? `onClose` is always required

Reply:
I'm not sure why it's there but I'll make the change!